util/xsl/826.2.par.xsl

```diff
@@ -0,0 +1,85 @@
+<!--  This xsl:
```

Comment:
In a tool chain, all xsl are applied one after the other and the obtained test cases are simulated. I think you should commit only one xsl for the magnetic saturation, otherwise the parameters appear twice in the par files and the tests will fail. @gautierbureau correct me if I am wrong

Reply:
I think that you should also add a test checking that the parameters are not already there

Reply:
use the command nrt-xsl to check ;)